umbraco-cloud/set-up/manage-subscriptions/new-shop.md

```diff
@@ -7,45 +7,49 @@ This article is only for customers in the payment system released on the 5th of
 For customers on Umbraco Cloud before the 5th of May, 2021 handling subscriptions is done through [the shop on Umbraco.com](https://shop.umbraco.com/profile/sign-in?returnURL=%2fprofile).
 {% endhint %}
 
-In this article, you will be able to find information on how to manage your subscriptions, download and pay invoices, and change your credit card on Umbraco Cloud.
+In this article, you will be able to find information on the following:
+- How to manage your subscriptions 
+- Download and pay invoices
+- And change your credit card on Umbraco Cloud.
 
 ## Manage Subscriptions
 
-To manage your subscription on Umbraco Cloud, go to the menu in the top right corner and select "Manage subscription".
+To manage your subscription on Umbraco Cloud, go to the menu in the top right corner and select "Organization".
 
-![manage subscriptions](images/manage-subscriptions-new.png)
+![manage subscriptions](images/org-view.png)
 
-You will see an overview of the subscriptions that you have running on your Umbraco Cloud account.
+You will see an overview of your organization on Umbraco Cloud. From here you can see the information about the organization.
 
-Here you can find information about the projects you are currently paying for on Umbraco Cloud.
+To see the subscribtions running under your organization click on "Subscriptions" in the side menu
 
 ![Your subscriptions](images/subscriptions.png)
 
 ## Payment methods
 
-To change your payment method on Umbraco Cloud, go to your profile and select "Payment Methods" in the left side menu.
+To change your payment method on Umbraco Cloud, go to your organization and select "Payment Methods" in the left side menu.
 
 ![Select Payment Methods](images/select_payment.png)
 
 On this page you can see the credit cards you have already added or you can add a new one.
 
-Once a credit card has been added it will show up in a drop-down when creating new projects and you can change the payment method for a specific project.
+Once a credit card has been added it will show up in a drop-down when creating new projects. You can also change the payment method for a specific project from here.
 
 ![Select Payment Methods](images/Payment_methods.png)
 
 ## Payment and Invoices
 
 On Umbraco Cloud, we are sending out one single invoice with all the projects that you are paying for via email every month.
 
-If you want to watch the Payment history and download an invoice from a previous month, you do this from the Payment setting in the dropdown menu on your projects.
+You can watch the invoices for your projects under your organization in the Payment history section. From here you can see the following for each invoice:
 
-![Payment setting](images/payment-settings.png)
+* The payment number
+* The total amount paid
+* When the invoice was created 
+* The due date
+* The status of the invoice
+* Download the invoice
```

Comment:
This is not really something you can see, like with the other items on the list.

Perhaps this should be moved out of the list, and instead be mentioned in a sentence below. Possibly together with the final sentence in the article.

Reply:
@sofietoft  Makes sense to me, I've moved it out and added it under the image.